lib/daos/userDao.js

```diff
@@ -8,6 +8,24 @@ const attributes = [
     'oauth_client_id'
 ];
 
+/**
+ * Creates a user based on the data the user provides
+ * @param {object} data the information provided by the user
+ * @returns the newly created user
+ */
+export const createOneUser = async data => {
+    const { firstName, lastName, email, id, oauth_client_id } = data;
+    const newUser = await users.create({
```

Comment:
Since the `newUser` is variable is being used, you can return the `users.create` without storing it in a variable